test/test-tcp-connect-error-busy.c

```diff
@@ -0,0 +1,62 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
I think you can use libuv contributors here instead of "Joyent, Inc. and other Node contributors"